boxoffice/assets/js/models/util.js

```diff
@@ -31,6 +31,11 @@ export const Util = {
   },
   getElementId: function(htmlString) {
     return htmlString.match(/id="(.*?)"/)[1];
+  },
+  getFormTemplate: function(htmlString, submitHandler) {
```

Comment:
Isn't `registerSubmitHandler` (removed below) simpler and easier to understand? Why is this better?